coalib/processes/BearRunning.py

```diff
@@ -27,8 +28,7 @@ def send_msg(message_queue, timeout, log_level, *args, delimiter=' ', end=''):
     :param end:           It is the value placed at the end of the message.
     """
     output = str(delimiter).join(str(arg) for arg in args) + str(end)
-    message_queue.put(LogMessage(log_level, output),
-                      timeout=timeout)
+    logging.log(log_level, output)
```

Comment:
Should I also deprecate `message_queue` as it of no use after deprecation of LogPrinterThread?

Reply:
yup, anything unused should be deprecated.

Reply:
@jayvdb done